tests/installation/common_criteria_configuration/common_criteria_encryption_pwd.pm

```diff
@@ -0,0 +1,19 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: The test module configure the disk encryption password
+# when using the Common Criteria role.
+# Maintainer: QE Security <none@suse.de>
+
+use parent 'y2_installbase';
+use strict;
+use warnings;
+
+sub run {
+    my $common_criteria_configuration_page = $testapi::distri->get_common_criteria_configuration();
```

Comment:
just `$common_criteria_configuration`, but not page, you are using the "controller" not the "page".

Reply:
done